subgraph-js/src/__tests__/printSubgraphSchema.test.ts

```diff
@@ -130,7 +130,7 @@ describe('printSubgraphSchema', () => {
 
       type User @key(fields: \\"id\\") @tag(name: \\"from-reviews\\") {
         id: ID!
-        username: String @external @tag(name: \\"on-external\\")
+        username: String @external
```

Comment:
Do we want another test to make sure that the old case errors?  

Edit: Never mind, I see the test in `compose.test.ts`